docs/en/observability/monitor-logs.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[monitor-logs]]
+= Monitor logs
+
+The {logs-app} in {kib} enables you to search, filter, and tail all your logs
+ingested into {es}. Instead of having to log into different servers, changing
+directories and tailing individual files, all the logs are available in the {logs-app}. 
```

Comment:
```suggestion
ingested into {es}. Instead of having to log into different servers, change
directories, and tail individual files, all your logs are available in the {logs-app}. 
```